pkg/minishift/provisioner/buildroot_provisioner.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Shouldn't it be Red Hat license?

Reply:
I think this is partially based on https://github.com/kubernetes/minikube/blob/master/pkg/provision/buildroot.go

Reply:
yes it's based on kubernetes buildroot provision